ajax/libs/amplitudejs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "amplitudejs",
+  "description": "A JavaScript library that allows you to control the design of your media controls in your webpage -- not the browser. No dependencies (jQuery not required) https://521dimensions.com/open-source/amplitudejs",
+  "filename": "amplitude.min.js",
+  "author": "521 Dimensions (https://521dimensions.com)",
+  "homepage": "http://jnordberg.github.io/gif.js/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/521dimensions/amplitudejs.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "webaudio",
+    "html5",
+    "javascript",
+    "audio-player"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/521dimensions/amplitudejs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "amplitude*"
```

Comment:
Les't use `**/*`
Because we usually use `**/*` pattern in `dist` basePath.
Thank you.